web/static/apps/wos/wos.js

```diff
@@ -0,0 +1,38 @@
+// TODO not sure about how to get the server_address properly
+const server_address = "http://localhost:8081";
+
+async function display_wall() {
+    let response = await fetch(server_address + "/wos/users");
```

Comment:
The API is hosted on the same server as the web site so you can just omit the `server_address` and the URL will resolve relative to the protocol/domain/port combo that the page was loaded on. So it would just be:

```js
let response = fetch("/wos/users");
```

However, see the comment later about renaming that endpoint.